thundra/plugins/invocation/invocation_plugin.py

```diff
@@ -24,48 +27,103 @@ def before_invocation(self, data):
         context = data['context']
         memory_size = getattr(context, 'memory_limit_in_mb', None)
         self.start_time = time.time() * 1000
+
+        function_name = getattr(context, constants.CONTEXT_FUNCTION_NAME, None)
+
+
+        self.start_time = int(time.time() * 1000)
+        active_span = self.tracer.get_active_span()
+
         self.invocation_data = {
             'id': str(uuid.uuid4()),
-            'transactionId': data['transactionId'],
-            'applicationName': getattr(context, 'function_name', None),
+            'type': "Invocation",
+            'agentVersion': '',
```

Comment:
Even though this can't be detected automatically, we can get this from constants and the constant is updated manually for every release

Reply:
Can start with `2.0.0` for the first release with new data model